sh.ppy.osu.appdata.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
```

Comment:
Spec recommends using `desktop-application` instead of `desktop` type in new metainfo files.